test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/accounts/SubscribedChangeAddressControllerSpec.scala

```diff
@@ -114,9 +115,9 @@ class SubscribedChangeAddressControllerSpec
 
       behave like redirectToStartBehaviour(performAction)
 
-      behave like displayEnterUkAddressPage(UserType.Individual, performAction)
-      behave like displayEnterUkAddressPage(UserType.Agent, performAction)
-      behave like displayEnterUkAddressPage(UserType.Organisation, performAction)
+      behave like displayEnterUkAddressPage(UserType.Individual, Self, performAction)
```

Comment:
it would make more sense if you were able to pass in `None` instead of `Self` here since `IndividualUserType` doesn't name sense in contexts outside of a return